bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -437,13 +426,13 @@ public boolean start() throws IOException {
       if (!Bytes.equals(startKey, endKey) && scanEndKey.length == 0) {
         splits.add(new SourceWithKeys(startKey, endKey, 0));
       }
-      LOG.info("Creating {} splits.", splits.size());
-      LOG.debug("Created splits {}.", splits);
+      SOURCE_LOG.info("Creating {} splits.", splits.size());
+      SOURCE_LOG.debug("Created splits {}.", splits);
       return splits;
     }
 
     /**
-     * Is the range of the region within the range of the scan?
+     * Checks if the range of the region is within the range of the scan.
```

Comment:
I don't think it is entirely contains. It is within the scan range. @sduskis , can you confirm?